index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
```

Comment:
[OPTIONAL] You have done well inserting a title :+1:. but the title should always reflect the page content. I think It will look professional if you update your title to something link :arrow_down: 

` <title>Hello Microverse Project</title>`